src/jobs/integration-test.yaml

```diff
@@ -26,7 +26,7 @@ parameters:
     description: "Path to kind config file."
     type: string
   kubernetes-version:
-    default: "v1.19.4"
+    default: "v1.20.2"
```

Comment:
We're using 1.20.5 in MCs already, so maybe worth setting this to the latest

Reply:
It's a shame but they don't create kind images for all releases. So this is the latest that is available.

https://hub.docker.com/r/kindest/node/tags?page=1&ordering=last_updated